tutorials/quickstart/tutorial.md

```diff
@@ -237,43 +237,44 @@ As shown in the diagram, we'll be adding a new messaging endpoint called **Shipp
+```
 
-Creating a **Console App (.NET Framework)** project which uses the older `*.csproj` file syntax will work just fine, but will look slightly different in Visual Studio, with nested **Properties**, **References**, and **packages.config** items instead of **Dependencies**.
 }}
 
-Depending on your environment, Visual Studio may create the project using C# 7.0. Let's change it to at least C# 7.1 so that we can use nice features like [an async Main method](https://blogs.msdn.microsoft.com/mazhou/2017/05/30/c-7-series-part-2-async-main/):
+First we'll create the **Shipping** project and set up its dependencies.
 
-1. In the **Solution Explorer**, right-click on the **Shipping** project and choose **Properties**.
-1. Switch to the **Build** tab.
-1. Under the **Output** heading, click the **Advanced…** button in the far lower-right corner.
-1. Change **Language version** to **C# latest minor version (latest)**.
-1. Click **OK**.
-1. Save and close the **Shipping** properties page.
+NOTE: In this tutorial, we'll use terminal commands like [`dotnet new`](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new), [`dotnet add package`](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-add-package), and [`dotnet add reference`](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-add-reference), but you can do the same things using the graphical tools in Visual Studio if you prefer.
 
-![Change Language Version](change-language-version.png "width=680")
+First let's make sure we're in the root of the project, where the **RetailDemo.sln** file is located:
 
-Now, we need to add references to the Messages project, as well as the NServiceBus, NServiceBus.Heartbeat, and NServiceBus.Metrics.ServiceControl packages
+```shell
+> cd tutorials-quickstart
+```
 
-1. In the newly-created **Shipping** project, add the `NServiceBus`, `NServiceBus.Heartbeat`, and `NServiceBus.Metrics.ServiceControl` NuGet packages, which are already present in other projects in the solution. In the Package Manager Console window, enter the following commands:
-    ```
-    Install-Package NServiceBus -ProjectName Shipping
-    Install-Package NServiceBus.Heartbeat -ProjectName Shipping
-    Install-Package NServiceBus.Metrics.ServiceControl -ProjectName Shipping
-    ```
-1. In the **Shipping** project, add a reference to the **Messages** project, so that we have access to the `OrderPlaced` event.
+Next, we'll create a new Console Application project named **Shipping**:
+
+```shell
+> dotnet new console --name Shipping --framework netcoreapp3.1
+```
+
+Now, we need to add references to the **Messages** project, as well as the NuGet packages we will need.
+
+```shell
+dotnet add Shipping reference Messages
+
+dotnet add Shipping package NServiceBus
```

Comment:
Should we specify versions on these?  Do we expect all of this to stay working properly when we transition to v8?

Reply:
It should be a task on V8 launch to update the quick start. For tutorials we only care about latest, so yeah I would want everyone to get whatever is latest at that moment.

Even if that didn't happen, with our wire compatibility it should Just Work®.